diff --git a/src/Event/Event.js b/src/Event/Event.js
index 1f7185f1bcb013b0f74dedb6b8f129071e93b565..6035a733304c5997144e5357ebb76fffbe4928eb 100644
--- a/src/Event/Event.js
+++ b/src/Event/Event.js
@@ -229,17 +229,11 @@ const Event = ({
     .enabled(!!onPress && !event.disablePress)
     .runOnJS(runGesturesOnJS)
     .withTestId(`pressGesture-${event.id}`)
-    .onTouchesDown(() => {
-      isPressing.value = true;
-    })
     .onEnd((evt, success) => {
       if (success) {
         runOnJS(onPressWrapper)();
       }
     })
-    .onFinalize(() => {
-      isPressing.value = false;
-    });
 
   const composedGesture = Gesture.Race(
     wrappedDragGesture,
@@ -325,9 +319,6 @@ const Event = ({
         accessibilityHint={`Show event ${event.id}`}
         style={[
           styles.container,
-          {
-            backgroundColor: event.color || DEFAULT_COLOR,
-          },
           containerStyle,
           event.style,
           animatedStyles,
diff --git a/src/Header/Header.js b/src/Header/Header.js
index f3e86bd77f342cf586d9f2fba79609d478793a6f..5a12b9490c04c2693e1d4d721a7e4b01a3f237be 100644
--- a/src/Header/Header.js
+++ b/src/Header/Header.js
@@ -69,7 +69,7 @@ const WeekViewHeader = ({
   currentDate,
   allDayEvents,
   initialDates,
-  formatDate,
+  formatDate = 'MMM D',
   style,
   textStyle,
   eventContainerStyle,
@@ -171,8 +171,4 @@ WeekViewHeader.propTypes = {
   updateCellsBatchingPeriod: PropTypes.number,
 };
 
-WeekViewHeader.defaultProps = {
-  formatDate: 'MMM D',
-};
-
 export default WeekViewHeader;
diff --git a/src/NowLine/NowLine.js b/src/NowLine/NowLine.js
index 913805eaefe114399aa241d008b68e15eafeb28a..1b7f3fbefbae8f4ee66d8719b1e377b6cd52c747 100644
--- a/src/NowLine/NowLine.js
+++ b/src/NowLine/NowLine.js
@@ -28,7 +28,7 @@ const useMinutesNow = (updateEvery = UPDATE_EVERY_MILLISECONDS) => {
   return minutesInDay(now);
 };
 
-const NowLine = ({ beginAgendaAt, color, width }) => {
+const NowLine = ({ beginAgendaAt = 0, color = '#e53935', width}) => {
   const { verticalResolution } = useVerticalDimensionContext();
   const minutesNow = useMinutesNow();
 
@@ -69,9 +69,4 @@ NowLine.propTypes = {
   color: PropTypes.string,
 };
 
-NowLine.defaultProps = {
-  color: '#e53935',
-  beginAgendaAt: 0,
-};
-
 export default React.memo(NowLine);
diff --git a/src/utils/HorizontalScroll.js b/src/utils/HorizontalScroll.js
index d8dd23b047ad2cc2a5405b69991a93a34e4dcde3..ba83767ca5341b1728549b39464824efca54af10 100644
--- a/src/utils/HorizontalScroll.js
+++ b/src/utils/HorizontalScroll.js
@@ -36,7 +36,7 @@ export const HorizontalSyncFlatList = React.forwardRef(
       () => gridScrollOffset.value,
       (newScrollOffset) => {
         if (headerRef) {
-          scrollTo(headerRef, newScrollOffset, 0, true);
+          scrollTo(headerRef, newScrollOffset, 0, false);
         }
       },
     );
