diff --git a/AnySizeDragSortableView.js b/AnySizeDragSortableView.js
index 9450cbc6782441c62fcb950491eca921cce72aee..ff0692677f25953c6fedf6d53066355b85be0114 100644
--- a/AnySizeDragSortableView.js
+++ b/AnySizeDragSortableView.js
@@ -6,11 +6,11 @@ import {
   View,
   PanResponder,
   LayoutAnimation,
-  Platform
+  Platform,
+  UIManager
 } from 'react-native';
 const PropTypes = require('prop-types')
 const ANIM_DURATION = 300
-const { UIManager } = NativeModules;
 
 if (Platform.OS === 'android') {
   if (UIManager.setLayoutAnimationEnabledExperimental) {
